forge-game/src/main/java/forge/game/card/Card.java

```diff
@@ -4661,6 +4661,9 @@ public final SpellAbility getSpellAbilityForStaticAbility(final String str, fina
         SpellAbility result = storedSpellAbility.get(stAb, str);
         if (result == null) {
             result = AbilityFactory.getAbility(str, this, stAb);
+            // apply text changes from the statics host
+            result.setIntrinsic(true);
+            result.changeTextIntrinsic(stAb.getChangedTextColors(), stAb.getChangedTextTypes());
```

Comment:
what part does need to set it to intrinsic?

i mean can't you just pass a parameter to the changeText function to overwrite it?

Reply:
yea I can try that, just wasn't sure how much I would need to pass that down recursively